apps/src/applab/designElements/elementUtils.js

```diff
@@ -103,7 +103,8 @@ var ELEMENT_ID_DENYLIST = [
   'submitButton',
   'unsubmitButton',
   'turtleImage',
-  'prompt-icon'
+  'prompt-icon',
+  'bubble'
```

Comment:
[i can't comment on the actual line, so pasting it into this comment]

```javascript
// TODO(dave): remove denylist once element ids inside divApplab
// are namespaced: https://www.pivotaltracker.com/story/show/113011395
```

is this tracked in jira?

Reply:
filed to track: https://codedotorg.atlassian.net/browse/STAR-1371